Package.swift

```diff
@@ -1,9 +1,9 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.9
 import PackageDescription
 
 let package = Package(
     name: "Zephyr",
-    platforms: [.iOS(.v11), .tvOS(.v11), .watchOS("9.0")],
+    platforms: [.iOS(.v12), .tvOS(.v12), .watchOS(.v9), .visionOS(.v1)],
```

Comment:
When I bumped to `swift-tools-version:5.9`, I got a deprecation warning:

<img width="1025" alt="Screenshot 2024-03-19 at 8 20 24 am" src="https://github.com/ArtSabintsev/Zephyr/assets/73652/61388421-8951-401d-81f8-20a559fe8f2a">

To make this go away I bumped iOS and tvOS to v12, but I'm not sure if that will have any unintended side effects on consumers.